homework-submission/todo-app/src/App.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import "./App.css";
+import Button from "./component/button.jsx";
+import Input from "./component/todoInput";
+
+const App = () => {
+  const [todos, setTodos] = useState([
+    {
+      id: 1,
+      description: "Get out of bed",
+      deadline: "2017-09-11",
+      done: true
+    },
+    {
+      id: 2,
+      description: "Brush teeth",
+      deadline: "2017-09-10",
+      done: false
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+      deadline: "2017-09-09",
+      done: false
+    }
+  ]);
+
+  const [description, setDescription] = useState("");
+  const [date, setDate] = useState("");
+
+  const handleInput = e => {
+    e.preventDefault();
+    e.target.type === "text"
+      ? setDescription(e.target.value)
+      : setDate(e.target.value);
+  };
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    const newItem = {
+      id: todos.length + 1,
+      description: description,
+      deadline: date,
+      done: false
+    };
+    const newTodos = [...todos, newItem];
+
+    setTodos(newTodos);
+    setDescription("");
+    setDate(null);
+    const inputText = document.querySelector("#text");
+    const inputDate = document.querySelector("#date");
```

Comment:
This is very illegal in React! Never interact directly with the DOM. You already have a state for description and date, use this value as value for your input field to manage their value using React (and to be able to reset them).

Reply:
fixed.